src/fosslight_scanner/fosslight_scanner.py

```
@@ -62,6 +63,29 @@
 ]
 
 
+def remove_tree(path):
+    if not os.path.exists(path):
+        return True
+    try:
+        shutil.rmtree(path)
+        return True
+    except Exception as ex:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)